composer.json

```diff
@@ -23,5 +23,9 @@
         }
     },
     "minimum-stability": "dev",
-    "license": "MIT"
+    "license": "MIT",
+    "require-dev": {
+        "doctrine/doctrine-bundle": "^1.6",
```

Comment:
dit klopt niet. de solr-bundle required nu doctrine. require-dev word alleen geinstalleerd als je de composer van de solr-bundle aanroept. je kan het liever in de require zelf zetten. want de functionaliteit die je toevoegd werkt niet zonder doctrine-bundle. als je in een installatie alleen de solr-bundle installeert moeten alle functies kunnen werken maar op een productie omgeving worden de require-dev's van een package nooit geinclude.